RFC-009-migrate-to-gitlab.md

```diff
@@ -0,0 +1,143 @@
+# Migrate Good Docs Project Git repositories to GitLab
+
+## Proposed by
+
+Alyssa Rock - https://thegooddocs.slack.com/team/U012KCMPP0V
+
+Co-sponsored by:
+
+- Bryan Klein
+- Deanna Thompson
+
+## Current status
+
+See the status indicated on the pull request description.
+
+
+## Proposal overview
+
+The Good Docs Project should migrate its Git repositories from GitHub to GitLab.
+Although the migration would involve some initial disruption to our community's workflow, the many advantages that GitLab offers would make the move worth it in the end.
+
+
+## Motivation
+
+Migrating our project to GitLab would involve many significant advantages, as explained in the following sections.
+
+### Ease of use and improved review workflows

```

Comment:
I'm going to make some suggestions to re-arrange this section because I think you buried the lead.
The feature parity plus benefits is the clincher. I think you should mention the "other members" last, because it's a supporting point but instantly alienated me, a non-user of GitLab.

Reply:
@flicstar , thanks so much! I've committed your suggestions!